### DISCUSSION TITLE
Recommendations for Profile Updates

### DISCUSSION THREAD

**Discussion by FlorianTolk** [gaia-x-dases/xapi-virtual-classroom#32] [General]
Good Afternoon,

While reviewing this profile for Verification on the [Profile Server](https://profiles.adlnet.gov), The ADL Initiative team did come across a few findings.

The templates for `muted`, `unmuted`, `started camera`, `stopped camera`, `shared`, and `unshared` have language that may be vague enough to cause confusion. Is it possible to either restructure the statement to make the actor be the person who performed the interaction, and the object be the actor that was muted/unmuted/etc, or change the language in the description of the template to make it clear the actor is the person who either performed the verb, or had the verb performed on them.

Additionally, we were not able to find the `Webinar` ActivityType used in a template, can it's use be expanded upon in your profile?

Otherwise the rest of this profile looks great, and we look forward to validating your work in the near future!

**Comment by quitterie-lcs** [comment]
Hi, Thanks for reviewing our profile, your comments are constructive for our first publication. It has helped us to rethink this part of the profile  ! :pray: 

We have taken into account the confusion caused by the `interacted` verb for the listed actions and will replace it by the action it self as a verb. For this we will define new concepts in this profile. 
We have decided to keep the person who is concerned with the action as the actor, whether the action is active or passive. Hence, when a third party runs of the action on a participant, this information will be tracked in a context field (for now, we agree on the `instructor` field). 

We will detail the documentation of each concerned statement to make this information clear for the reader.

About `Webinar` ActivityType, is it the concept that is linked in the profile on the ADL server ? 

Thank you !

**Comment by FlorianTolk** [comment]
Good Afternoon,

Keeping the person who is affected by the action the actor is a great idea, and will certainly help prevent any misinterpretations of the rules as written in your profile.

However, when you are defining the person who actually pressed the mute button, we would strongly suggest defining a context extension instead of using the instructor context property. The instructor is meant to only be the actor that is running the course, and if a moderator were to mute a person it would go against the spirit of this property.

The Webinar ActivityType is from the entry from the profile server. We do see it as having value in this profile, but its use should be fleshed out in a template. This post was in response to the request to have this profile verified. We are very eager to do so, but want to ensure it is in the best possible state first.

**Comment by quitterie-lcs** [comment]
Good afternoon, 

According to your recommendation, instructor won't be used for actions where we wanted to use it. But, I would like to get your opinion on what I think would be a proper way to modelise this activities. The problem was that it defined new verbs for actions that already exist in xAPI vocabular and those new verbs would be very specific and unreusable.

Hence, for following statements: muted, unmuted, started camera and stopped camera, [started verb](https://profiles.adlnet.gov/profile/9005d7f0-8ca8-4714-baf7-06ca5301333e/concepts/553af7a8-f718-4e82-a830-db71ea4c428c) and newly created `stopped` verb would be used with newly created activity types `micro` and `camera` as objects in the templates.

For shared screen and unshared screen, we follow the same logic: [share](https://profiles.adlnet.gov/profile/9005d7f0-8ca8-4714-baf7-06ca5301333e/concepts/17f9320d-9428-41e3-90c7-11b1ee291033) and [unshared](https://profiles.adlnet.gov/profile/9005d7f0-8ca8-4714-baf7-06ca5301333e/concepts/48802b0d-fe68-4d7a-a45b-f5bc7154d625) verb and a newly created activity type `screen` as the object

And for raised and lowered hand, it would be slightly different: we would consider that this action is actually a reaction to the virtual classroom. So that the verb would be the newly created `reacted` verb and the object the virtual classroom itself, and a newly created `result emoji extension` containing the unicode of the `hand raised` emoji. The lowered hand would be this action but with the newly created `unreacted` verb and the same object and result fields.

Furthermore, regarding the session ID extension, is there one of the both defined that is better to use ? As one is from the CMI5 profile and the other was specifically created for the video activity type but its definition could fit to other activity types. 

Finally, the problem with the webinar activity type on the server is that we had linked this activity type to our profile when creating but finally decided not to use it. However, when trying to remove the link, we got the following error from the server. I previously open a ticket on the Jira from the server but it seems that it has been deleted 

> concepts Error
> 
> ERROR_REMOVE_CONCEPT_LINK Unauthorized

Thanks for your help and advice!

**Comment by quitterie-lcs** [comment]
Hi @FlorianTolk, I would like to know if the verification of this profile for the ADL server is still in process.

Best regards

Quitterie Lucas
